linux_new/Jenkinsfile

```diff
@@ -779,9 +800,46 @@ stage('Build & Archive Package') {
             echo "Exception in build for ${DistArrayElement}: ${ex}"
             currentBuild.result = 'FAILURE'
         } finally {
-            archiveArtifacts artifacts: '**/build/ospackage/*,**/build/reports/**,**/packageTest/dependencies/deb/*',
-            onlyIfSuccessful: false, allowEmptyArchive: false
-            }
+
+          def basePattern = '**/build/ospackage/*,**/build/reports/**'
```

Comment:
Please indent to match the replaced code.

Reply:
should be sorted, if my ide is to be believed :)

Reply:
Here's an easy c+p for the indentation.

[snippet.txt](https://github.com/user-attachments/files/20161936/snippet.txt)